### ISSUE TITLE
Crash when placing down a cabinet.

### ISSUE THREAD

**Issue by ExDrill** [rotgruengelb/MyNethersDelightRefabricated#7]
```
java.lang.IllegalStateException: Invalid block entity farmersdelight:cabinet // vectorwing.farmersdelight.common.block.entity.CabinetBlockEntity state at class_2338{x=43, y=4, z=52}, got Block{mynethersdelight:blackstone_bricks_cabinet}[facing=south,open=false]
	at knot//net.minecraft.class_2586.method_61175(class_2586.java:57)
	at knot//net.minecraft.class_2586.<init>(class_2586.java:51)
	at knot//net.minecraft.class_2624.<init>(class_2624.java:33)
	at knot//net.minecraft.class_2621.<init>(class_2621.java:25)
	at knot//vectorwing.farmersdelight.common.block.entity.CabinetBlockEntity.<init>(CabinetBlockEntity.java:57)
	at knot//net.minecraft.class_2591.method_11032(class_2591.java:99)
	at knot//vectorwing.farmersdelight.common.block.CabinetBlock.method_10123(CabinetBlock.java:100)
	at knot//net.minecraft.class_2818.method_12010(class_2818.java:286)
	at knot//net.minecraft.class_1937.method_30092(class_1937.java:232)
	at knot//net.minecraft.class_638.method_30092(class_638.java:160)
	at knot//net.minecraft.class_1937.method_8652(class_1937.java:217)
	at knot//net.minecraft.class_1747.method_7708(class_1747.java:153)
	at knot//net.minecraft.class_1747.method_7712(class_1747.java:76)
	at knot//net.minecraft.class_1747.method_7884(class_1747.java:46)
	at knot//net.minecraft.class_1799.method_7981(class_1799.java:373)
	at knot//net.minecraft.class_636.method_41934(class_636.java:363)
	at knot//net.minecraft.class_636.method_41933(class_636.java:318)
	at knot//net.minecraft.class_636.method_41931(class_636.java:279)
	at knot//net.minecraft.class_636.method_2896(class_636.java:317)
	at knot//net.minecraft.class_310.method_1583(class_310.java:1852)
	at knot//net.minecraft.class_310.method_1508(class_310.java:2111)
	at knot//net.minecraft.class_310.method_1574(class_310.java:1938)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1246)
	at knot//net.minecraft.class_310.method_1514(class_310.java:882)
	at knot//net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

**Comment by ChrysanthCow** [comment]
This is an issue with a 1.21.1 change where it'll validate the blocks.

I'll have to add this cabinet as a supported block.

**Comment by ChrysanthCow** [comment]
Will be fixed in 1.1.1
